src/components/main-page/Registration/Registration.tsx

```diff
@@ -22,6 +25,11 @@ export function Registration() {
         {t("registration.title")}
       </Heading>
       <HackathonTimer />
+      <div className={styles.linksContainer}>
```

Comment:
What do you think about `styles.buttonsContainer`  className? Maybe it will describe better this element

Reply:
Adjusted accordingly in https://github.com/akai-org/akai-code/pull/44/commits/1e73bfcc3a60dfe57d8048d6a537323d18ce4b8e